adal/src/main/java/com/microsoft/aad/adal/AcquireTokenSilentHandler.java

```diff
@@ -418,6 +418,7 @@ private AuthenticationResult acquireTokenWithCachedItem(final TokenCacheItem cac
      * Old version of ADAL doesn't mark token stored in regular RT entry as MRRT even it is. The logic to look for
      * MRRT is when RT is not found or found RT is also MRRT. To support the old behavior, do a separate check on
      * the existence for MRRT token entry.
+     * just testing
```

Comment:
Any specific reason to have these tags in the merge PR?

Reply:
Forgot to remove this, already purged